src/Compiler/Driver/CompilerOptions.fsi

```diff
@@ -52,7 +55,23 @@ val FilterCompilerOptionBlock: (CompilerOption -> bool) -> CompilerOptionBlock -
 /// Parse and process a set of compiler options
 val ParseCompilerOptions: (string -> unit) * CompilerOptionBlock list * string list -> unit
 
-val DisplayBannerText: TcConfigBuilder -> unit
+val DisplayBannerText:
+    tcConfigB: TcConfigBuilder
+    -> print: (string -> unit)
+    -> unit
+
+val DisplayHelpFsc: 
+    tcConfigB: TcConfigBuilder 
+    -> blocks: CompilerOptionBlock list 
+    -> print: (string -> unit) 
+    -> exit: (unit -> 'a)
```

Comment:
Please use `'T` in signature file.  Though we don't do that everywhere.

Reply:
Maybe this doesn't even need a type parameter and can just be `unit`?

Reply:
Not relevant anymore